.travis.yml

```diff
@@ -7,20 +7,21 @@ go:
     - tip
 
 env:
+    global:
+        - TRAVIS_ENV="-e TRAVIS=$TRAVIS
+                      -e CI=$CI
+                      -e TRAVIS_COMMIT=$TRAVIS_COMMIT
+                      -e TRAVIS_COMMIT_RANGE=$TRAVIS_COMMIT_RANGE
+                      -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG
+                      -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST
+                      -e TRAVIS_PULL_REQUEST_SHA=$TRAVIS_PULL_REQUEST_SHA
+                      -e TRAVIS_PULL_REQUEST_SLUG=$TRAVIS_PULL_REQUEST_SLUG
+                      -e TRAVIS_BRANCH=$TRAVIS_BRANCH
+                      -e TRAVIS_JOB_ID=$TRAVIS_JOB_ID
+                      -e TRAVIS_BUILD_DIR=$TRAVIS_BUILD_DIR"
     - BUILDAH_ISOLATION=oci
     - BUILDAH_ISOLATION=chroot
     - BUILDAH_ISOLATION=rootless
```

Comment:
I couldn't find a Travis run for this PR.  Based on the [docs for global variables](https://docs.travis-ci.com/user/environment-variables#global-variables), I think these three `BUILDAH_ISOLATION` values need to be moved into a `matrix:` subsection alongside the `global:` one to get them to trigger builds.